File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.py
def solution(participant, completion):
    p = set(participant)
    s = set(completion)
    dict = {}
    
    for name in participant:
        dict[name] = dict.get(name,0) + 1
        
    for name in completion:
        dict[name] -= 1
    
    for name, cnt in dict.items():
        if cnt != 0:
            return name

File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.cpp
#include <bits/stdc++.h>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";
    unordered_map<string, int> mp;
    for (auto s : participant){
        mp[s] += 1;
    }
    
    for (auto s : completion){
        mp[s] -= 1;
    }
    
    for (auto i : mp){
        if (i.second != 0){
            answer += i.first;
            break;
        }
    }
    
    return answer;
}